### DISCUSSION TITLE
i want to render callout as tables but also render code blocks as itself

### DISCUSSION THREAD

**Discussion by Hyeonjun0527** [mvdkwast/obsidian-copy-as-html#53] [Q&A]
i want to render callout as tables but also render code blocks as itself.
but because of menu's limitation, we only could render both of callout and code blocks.
i want to seperate call out as tables menu and code blocks as tables menu.
then i'll sooooooooooooo happy. sorry for bad english. thank you.

**Comment by mvdkwast** [comment]
Hi @Hyeonjun0527, thanks for your reaching out, and sorry for the late response. I wonder what your use-case is, the feature was mainly intended as a work-around for pasting into google docs.

I'm converting this discussion to a feature request : #60

**Comment by mvdkwast** [comment]
Implemented in #60.
